### DISCUSSION TITLE
Asesoria y seguimiento

### DISCUSSION THREAD

**Discussion by carlos319** [rramosp/ai4eng.v1#292] [00 - General]
Buenas noches @Felipe-RA, espero estes muy bien, en mi grupo queremos realizar una pequeña reunion con tigo para finiquitar detalles de nuestra proxima entrega. ya nos dces que dia puedes de 6pm en adelante y gracias!

**Comment by Felipe-RA** [comment]
Saludos @carlos319 

Puedo echarles una mano el jueves a las 8 p.m

**Reply by carlos319** [reply]
ok. perfecto!!

**Reply by carlos319** [reply]
https://meet.google.com/tuz-ufgs-nwe

Buenas noches

**Reply by carlos319** [reply]
Muy buenas noches @Felipe-RA , gracias por el acompañamiento, hermano yo creo que ya estamos con la entrega, si tienes forma mañana para ver unos detalles se le agradeceria ya usted nos dice la hora y mil gracias nuevamente.

https://meet.google.com/tuz-ufgs-nwe
